kiliautoml/utils/helpers.py

```diff
@@ -78,8 +79,34 @@ def get_asset_memoized(
     total: Optional[int],
     skip: int,
     status_in: Optional[List[AssetStatusT]] = None,
+    asset_filter: Optional[AssetFilterArgsT] = None,
 ) -> List[Any]:
     kili.assets = backoff.on_exception(backoff.expo, exception=Exception, max_tries=3)(kili.assets)
+
+    additional_filtering_arguments: Dict[str, Any] = {}
+    if asset_filter is not None:
```

Comment:
using a dict comprehension would be a bit lighter: https://stackoverflow.com/questions/3420122/filter-dict-to-contain-only-certain-keys

Reply:
with the if else condition to log the warning, it is complicated to use dict comprehension; it feels like code is less readable, don't you think?

Reply:
ok good point